manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.7.0
+PackageLocale: en-US
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
+License: MPL-2.0
+ShortDescription: OneScript engine
+Tags:
+- "Console"
+- "Command-Line"
+- "OneScript"
+- "oscript"
```

Best practice is to use all lower case. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- "console"
- "command-line"
- "onescript"
- "oscript"
```